submissions/evgeniy24/friends_app/app.js

```diff
@@ -0,0 +1,234 @@
+const REQUEST_URL  = 'https://randomuser.me/api/?results=20',
+    FRIENDS_LIST_WRAP = document.querySelector('.friendsList-wrap'),
+    MENU_WRAP = document.querySelector('.menu-wrap'),
+    SORTING_MENU = document.querySelector('.filter-sort-menu'),
+    SEARCH_FIELD = document.querySelector('#search'),
+    SORT_BY_NAME_AZ = document.querySelector('.nameSortAZ'),
+    SORT_BY_NAME_ZA = document.querySelector('.nameSortZA'),
+    SORT_BY_AGE_YO = document.querySelector('.ageSortYoungerFirst'),
+    SORT_BY_AGE_OY = document.querySelector('.ageSortOlderFirst'),
+    FILTER_MENU = document.querySelector('.filter-gender'),
+    FILTER_BY_MALE = document.querySelector('.filter-gender__male'),
+    FILTER_BY_FEMALE = document.querySelector('.filter-gender__female'),
+    FILTER_BY_ALL = document.querySelector('.filter-gender__all'),
+    MOBILE_MENU_BUTTON = document.querySelector('.button-container');
+let friendsList = [],  
+    searchResultList,
+    sortedList,
+    errorButton,
+    genderNeutralList, filteredByGenderList,
+    scrollDepth;
+    
+const sendRequest = function(method, url) {
+    return fetch(url)
+        .then(handleErrors)
+        .then(function(response) {
+            return response.json();
+        });
+}
+
+function handleErrors(response) {
+    if (response.status > 400) {
+        throw Error(response.statusText);
+    }
+    return response;
+}
+
+function startApp() {
+    sendRequest('GET', REQUEST_URL)
+    .then( function(data) { 
+        friendsList = data.results;
+        renderFriendsList(friendsList);
+    })
+    .catch( function(err) {
+        renderErrorMessage();
+    });
+}
+
+startApp();
+
+// drawing a list of friends---------------------------
```

Comment:
Redundant comment. Please, remove all of them

Reply:
All comments have been removed!